v1.16/Gemfile

```diff
@@ -0,0 +1,64 @@
+source 'https://rubygems.org'
+
+gem 'oj', '3.14.2'
+gem 'activesupport', '6.1.7'
+gem 'http_parser.rb', '0.8.0'
+gem 'tzinfo', '>= 1.2.11'
+gem 'resolv', '0.2.2'
+gem 'json', '2.6.3'
+gem 'yajl-ruby', '1.4.1'
+gem 'async', '1.31.0'
+gem 'async-http', '0.60.1'
+gem 'fluentd', '1.16.2'
+gem 'prometheus-client', '4.1.0'
+gem 'bigdecimal', '3.1.4'
+gem 'webrick', '1.8.1'
+gem 'gelf', '3.1.0'
+gem 'elasticsearch', '8.6.0'
+gem 'elasticsearch-xpack', '7.17.7'
+gem 'snappy', '0.2.0'
+gem 'fluent-plugin-detect-exceptions', '0.0.15'
+gem 'fluent-plugin-remote-syslog', '1.1.0'
+gem 'fluent-plugin-webhdfs', '1.5.0'
+gem 'fluent-plugin-prometheus', '2.0.3'
+gem 'fluent-plugin-s3', '1.7.2'
+gem 'fluent-plugin-rewrite-tag-filter', '2.4.0'
+# gem 'fluent-plugin-azure-storage-append-blob', '0.2.1'

```

Comment:
no longer maintained by microsoft: https://github.com/microsoft/fluent-plugin-azure-storage-append-blob

Reply:
are the APIs compatible?

Reply:
it's a direct fork, so assume they are

Reply:
this is a no brainer then, let's switch to the new one